accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,378 @@
+    - `CreateSecurityTokenProvider(SecurityTokenRequirement)`
+    - `CreateSecurityTokenSerializer(SecurityTokenVersion)`
+* **CommunicationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **CommunicationObjectAbortedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **CommunicationObjectFaultedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **DuplexChannelFactory**
+    - `.ctor(InstanceContext, Binding, EndpointAddress)`
+    - `.ctor(InstanceContext, Binding, String)`
+    - `.ctor(InstanceContext, String)`
+    - `.ctor(InstanceContext, String, EndpointAddress)`
+* **DuplexClientBase**
+    - `.ctor(InstanceContext)`
+    - `.ctor(InstanceContext, String)`
+    - `.ctor(InstanceContext, String, EndpointAddress)`
+    - `.ctor(InstanceContext, String, String)`
+* **EndpointAddress**
+    - `ApplyTo(Message)`
+* **EndpointIdentity**
+    - `Equals(Object)`
+    - `GetHashCode()`
+* **EndpointNotFoundException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **FaultException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **InvalidMessageContractException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **MessageHeaderException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **MessageSecurityOverTcp**
+    - `get_ClientCredentialType()`
+    - `set_ClientCredentialType(MessageCredentialType)`
+* **NetHttpBinding**
+    - `CreateBindingElements()`
+* **NetHttpsBinding**
+    - `.ctor()`
+    - `.ctor(BasicHttpsSecurityMode)`
+    - `CreateBindingElements()`
+* **NetTcpBinding**
+    - `.ctor(String)`
+    - `CreateBindingElements()`
+* **ProtocolException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **QuotaExceededException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **ServerTooBusyException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **ServiceActivationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+
+### System.ServiceModel.Channels
+
+* **AddressHeader**
+    - `Equals(Object)`
+    - `GetHashCode()`
+* **MessageHeaders**
+    - `set_To(Uri)`
+* **SecurityBindingElement**
+    - `CreateSecureConversationBindingElement(SecurityBindingElement)`
+* **TransportSecurityBindingElement**
+    - `GetProperty<T>(BindingContext)`
+* **WebSocketTransportSettings**
+    - `get_DisablePayloadMasking()`
+    - `set_DisablePayloadMasking(Boolean)`
+
+### System.ServiceModel.Security
+
+* **MessageSecurityException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **SecurityAccessDeniedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **SecurityNegotiationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **X509ServiceCertificateAuthentication**
+    - `set_CertificateValidationMode(X509CertificateValidationMode)`
+
+### System.Threading
+
+* **EventWaitHandle**
+    - `OpenExisting(String)`
+    - `TryOpenExisting(String, EventWaitHandle)`
```

Comment:
I am not sure how to deal with the ctor:

it calls `CreateEventCore`:

https://github.com/dotnet/runtime/blob/ecdf9de474459f17efb90fd64609d164bcf454d4/src/libraries/System.Private.CoreLib/src/System/Threading/EventWaitHandle.cs#L22-L27

which throws on Unix `if name != null`

https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Private.CoreLib/src/System/Threading/EventWaitHandle.Windows.cs#L19-L23

Reply:
See above. We decided to mark those because these APIs work as long as name is null.